docs/intro/tutorial.rst

```diff
@@ -83,8 +83,8 @@ sites, we define fields for each of these three attributes. To do that, we edit
         link = scrapy.Field()
         desc = scrapy.Field()
 
-This may seem complicated at first, but defining the item allows you to use other handy
-components of Scrapy that need to know what does your item look like.
+This may seem complicated at first, but defining an item class allows you to use other handy
+components and helpers within Scrapy.
```

Comment:
Hi Elias,

Happy to help! :-) Sure. It is fixing a grammatical issue. Mainly the last bit of that sentence and I also tried to explain it in a way that made more sense. Well, to me that is

If it is really unclear and others feel the same way, you may want remove the word ‘does’ to fix the grammatical error in the current version.

Rick

> Op 26 aug. 2015, om 18:52 heeft Elias Dorneles <notifications@github.com> het volgende geschreven:
> 
> In docs/intro/tutorial.rst <https://github.com/scrapy/scrapy/pull/1449#discussion_r38005559>:
> 
> > @@ -83,8 +83,8 @@ sites, we define fields for each of these three attributes. To do that, we edit
> >          link = scrapy.Field()
> >          desc = scrapy.Field()
> >  
> > -This may seem complicated at first, but defining the item allows you to use other handy
> > -components of Scrapy that need to know what does your item look like.
> > +This may seem complicated at first, but defining an item class allows you to use other handy
> > +components and helpers within Scrapy.
> Hello @rhoekman <https://github.com/rhoekman> -- thanks for checking our docs!
> 
> I'm not sure if I understand what exactly this is fixing, could you explain? (non-native English speaker here, sorry if I'm missing something)
> 
> The idea here was to state that some Scrapy components need to know what your item looks like (namely, the item schema), I feel this note is being lost in this new version.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/scrapy/scrapy/pull/1449/files#r38005559>.
>